src/main/java/Duke.java

```diff
@@ -1,10 +1,77 @@
+import java.util.Objects;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you\n");
+
+        Scanner scan = new Scanner(System.in);
+        String input = scan.nextLine();
+        boolean isBye = false;
+        Task[] inputs = new Task[100];
+        for(int k = 0; k < 100; k++) {
+            inputs[k] = new Task("");
+        }
+
+        int count = 0;
+        int number = 1;
```

Comment:
Perhaps adding a comment to explain what the "number" variable represents would be helpful to other readers.